### PR TITLE
Add newsletter and Slack links to community page.

### PR DESCRIPTION
**Pull request by ilfrin**

### CODE REVIEW

**Comment by ilfrin** [comment]
@electrum @dain I thought this would be an interesting addition for the Presto community to enjoy.

Please review and merge.

File: community.html [hunk 1]
```diff
@@ -95,6 +95,21 @@ <h2>Twitter</h2>
             <a href="https://twitter.com/search?q=%23prestodb">#prestodb</a>
             hashtag on Twitter to keep up with the latest Presto news.
         </p>
+
+        <h2>Slack</h2>
+
+        <p>
+            The Presto developer community is coordinating activities via
+            <a href="https://prestodb.slack.com">prestodb.slack.com</a>
```
Comment (martint) [inline community.html]: We point to the self-signup link, instead:

```
https://join.slack.com/t/prestodb/shared_invite/enQtNDczNzEzMjYxMTc0LTc5MTVmZjE1N2E4YjJmZDBlNDBkOGUwMjM5YWIwM2E1ZjI3M2FiYzk2MTRmZjFjYjQ3ZmIwNzAzZTU5MWM4ZGU
```

Comment (martint) [reply community.html]: We might want to get rid of the IRC channel reference above, too, since no one is really using it.

Comment (electrum) [inline community.html]: Nit: end sentence in period

Comment (dain) [reply community.html]: Totally agree.  I don't even have an IRC client on my machine anymore.

Comment (ilfrin) [reply community.html]: link href changed, IRC removed

**Comment by ilfrin** [comment]
AC

**Pull request merged**